packages/matchbox/src/components/Modal/Modal.js

```diff
@@ -94,7 +94,13 @@ const Modal = React.forwardRef(function Modal(props, ref) {
             role="dialog"
             aria-modal="true"
           >
-            <Box p={['400', null, '700']} size="100%" ref={overlayRef} data-id="modal-overlay">
+            <Box
+              p={['400', null, '700']}
+              pt={['5rem', null]}
```

Comment:
should we use a token?

Reply:
Might have to create a new one. Our tokens jump from 4rem to 10rem and 4rem is too small and the modal is still hidden behind the app header, 10rem is pretty large but I guess we could use it. What do you think?

Reply:
Oh yea, i'd just leave it at 5rem then, doesn't make sense to make an `850` token at `5rem`